src/Prjkt/Component/Repofuck/Repofuck.php

```diff
@@ -307,13 +307,17 @@ protected function map(array $inserts, array $keys = [], $entity = null) : Model
 			throw new EntityNotDefined;
 		}
 
+		$fillable = $entity->getFillable();
+		
 		foreach($inserts as $key => $val)
 		{
 			if ( count($keys) > 0 && ! in_array($keys, $key) ) {
 				break;
 			}
 
-			$entity->{$key} = $val;
+			if (array_key_exists($key, $fillable)){
```

Comment:
`if ( count($keys) > 0 && ! in_array($keys, $key) ) ... break;`

As I read it: "If `$keys` has content and the `$keys` is not in the `$key`, `break.`

Have I misread the thing?

http://php.net/manual/en/function.in-array.php